java/src/apps/ActionListBundle_da.properties

```diff
@@ -88,7 +88,7 @@ jmri.jmrit.MemoryFrameAction = \u00c5bn Hukommelses Monitor
 jmri.jmrit.withrottle.WiThrottleCreationAction = Start WiK\u00f8rekontrol Server
 jmri.jmris.json.JsonServerAction = Start JMRI JSON Server
 jmri.jmris.srcp.JmriSRCPServerAction = Start JMRI SRCP Server
-jmri.jmris.simpleserver.SimpleServerAction = Start Simpel JMRI Server
+jmri.jmris.simpleserver.SimpleServerAction = Start JMIR Simpel Server
```

Comment:
Should that be JMRI?  We tend to stick to the English acronym in translations.

Reply:
Thank you for catching that.